python/tests/test_healthcheck.py

```diff
@@ -103,9 +105,9 @@ def test_GenerateModelReport(sample):
 def test_GenerateModelReport_Failing(sample_without_predictorbinning):
     with pytest.raises(Exception) as e_info:
         sample_without_predictorbinning.generate_model_reports(
-            name="MyOrg", modelid=["bd70a915-697a-5d43-ab2c-53b0557c85a0"]
+            name="MyOrg", model_list="bd70a915-697a-5d43-ab2c-53b0557c85a0"
```

Comment:
Well now you've made it model_list and are passing a string :)

Reply:
It is to make the test fail as the name suggests 😄

Reply:
well look at me not reading :D